domain-rrd/rrd-workflows/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -67,6 +60,11 @@ public AbdEvidence mapAnnotationsToEvidence(MasCollectionAnnotation masCollectio
       if (masDocument.getAnnotations() != null) {
         String documentId = masDocument.getEfolderversionrefid();
         String receiptDate = masDocument.getRecDate();
+        try {
+          receiptDate = formatter1.format(formatter.parse(receiptDate));
+        } catch (Exception e) {
+          log.error("Un-parsable date for ReceiptDate: {}.", receiptDate);
+        }
```

Comment:
Just log a message here? If it cannot handle it, this is probably not the right place to format the date string.

Reply:
On line 62, `receiptDate` is set to whatever comes from the `masDocument` so in the case of failure it will just stay as the recDate from MAS.

Reply:
I see that. It is confusing to format a date string here where it really doesn't care about the format. It is in the original code. It might be a good idea not to introduce much code change to avoid to bring new errors at this stage.

Reply:
I understand, the reason why I think it should happen here is we are receiving the date on a per document basis so it is easiest to reformat right where we read it in, instead of letting it trickle down to the python assessment where it would be reformatted on a per element basis.

Reply:
The PR does not have to go in right away, I just saw an opportunity to reduce the clutter downstream.